wrappers/obj-c/ODWLogger.mm

```diff
@@ -4,6 +4,7 @@
 //
 #include "ILogger.hpp"
 #import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
```

Comment:
This might not work for Mac

Reply:
UIKit (iOS) and IOKit (OSX) are both problematic for the shared cocoapod:  https://dev.azure.com/msblox/_git/CocoaPods?path=%2FSpecs%2FOneDsCppSdk%2F3.5.117.1%2FOneDsCppSdk.podspec

Reply:
created an API in `ODWLogManager` instead of listening to the notification in the SDK. The host app can call the API on `appWillTerminate` notification.